src/services/formatting/rules.ts

```diff
@@ -213,25 +213,15 @@ namespace ts.formatting {
         public NoSpaceBetweenYieldKeywordAndStar: Rule;
         public SpaceBetweenYieldOrYieldStarAndOperand: Rule;
 
-        // Async-await
+        // Async functions
         public SpaceBetweenAsyncAndFunctionKeyword: Rule;
-        public NoSpaceBetweenAsyncAndFunctionKeyword: Rule;
-        public SpaceAfterAwaitKeyword: Rule;
-        public NoSpaceAfterAwaitKeyword: Rule;

```

Comment:
What about the `NoSpaceAfterAwaitKeyword` one? Is that just encompassed by `SpaceAfterCertainKeywords`?